### ISSUE TITLE
how can i train fcnn if one window can have more than one label?

### ISSUE THREAD

**Issue by ankurgupta7** [rbgirshick/py-faster-rcnn#382]

**Comment by rtgoring** [comment]
Yes

**Comment by ankurgupta7** [comment]
How can i do that?

**Comment by rohitghosh** [comment]
Hi @ankurgupta7 ,

Can you elaborate a bit more on the pain ? Ideally, while specifying annotations in 'annotations.txt' you just mention the object class along with co-ordinates for top left and bottom-right of bounding boxes for each object.  Overlapping windows won't be an issue

**Comment by ankurgupta7** [comment]
Yeah, true. but i dont have a dataset that has all the labels for each image. i have some images with label a and some with b. ideally they all the images would have had labels a&b. if i train with such a dataset, loss function would penalize 'a' for images with label 'b'. because it kind of assumes that both the labels are mutually exclusive. Am i making any sense?
TL;DR my dataset is incomplete. how can i not penalize labels that 'should have been '  but 'are not' present in the trainig dataset?

**Comment by ankurgupta7** [comment]
@rohitghosh my question is essentially about converting the SoftmaxLoss in "loss_cls" layer to SigmoidCrossEntropy. fails due to mismatching size between {bottom: "cls_score",  bottom: "labels"}

**Comment by ycxia** [comment]
@ankurgupta7  Hi, did you solve this problem?
